### PR TITLE
Roll hyper to v0.10

### PR DESCRIPTION
**Pull request by mayah**
Since hyper v0.10, we need to use tls implementation for https access.
Use hyper-native-tls for that purpose.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -16,15 +16,16 @@ readme = "README.md"
...
-url = "1.1.0"
+chrono = "^0.3"
+hyper = "^0.10"
+hyper-native-tls = "^0.2"
+rustc-serialize = "^0.3"
+url = "^1.1"
 
 [dependencies.serde]
-version = "0.7.0"
+version = "^0.7"
```
Comment (programble) [inline Cargo.toml]: y tho

File: Cargo.toml [hunk 2]
```diff
@@ -16,15 +16,16 @@ readme = "README.md"
 default = ["serde"]
 
 [dependencies]
-chrono = "0.2.17"
-hyper = "0.9.0"
-rustc-serialize = "0.3.16"
-url = "1.1.0"
+chrono = "^0.3"
+hyper = "^0.10"
+hyper-native-tls = "^0.2"
```
Comment (programble) [inline Cargo.toml]: This should be a dev-dependency.

Comment (mayah) [inline Cargo.toml]: Done

File: Cargo.toml (hunk 1 above)
Comment (mayah) [inline Cargo.toml]: Done

**Comment by mayah** [comment]
PTAL

**Pull request merged**